.circleci/config.yml

```diff
@@ -3,12 +3,35 @@
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 version: 2
+
+job_defaults: &job_defaults
+  machine:
+    image: circleci/classic:latest
+  working_directory: ~/job-manager
+
+run_env_setup: &run_env_setup
+  run:
+    name: Setup Environment Variables
+    command: |
+      echo 'export PATH="${PATH}:${HOME}/job-manager/git-secrets"' >> $BASH_ENV
+
+run_secret_setup: &run_secret_setup
+  run:
+    name: Install git secrets and configure minnie-kenny.sh
+    command: |
+      git clone https://github.com/awslabs/git-secrets.git
+      pushd git-secrets >/dev/null
+      git checkout ad82d68ee924906a0401dfd48de5057731a9bc84
```

Comment:
That hash corresponds to the latest version, [`1.3.0`](https://github.com/awslabs/git-secrets/releases/tag/1.3.0).

One could in theory download / checkout the floating git-tag "1.3.0". But that's less secure because it could be changed by anyone with commit access to awslabs/git-secrets on GitHub.

Instead, unlike "1.3.0" that could be repointed at another commit, one can be ([reasonably](https://github.blog/2017-03-20-sha-1-collision-detection-on-github-com/)) guaranteed that `ad82d68ee924906a0401dfd48de5057731a9bc84` will always points to the underlying commit and code that we're looking for.